<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>
    Pipeline tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs
  </title>
<style>
#title{
  background-color: #ec407a;
  color: white;
  padding: 2em;
}
h1 {
  margin-bottom: 0em;
}
#subtitle {
  font-variant: small-caps;
  text-transform: lowercase;
  letter-spacing: 0.2em;
}
.run-info ul {
  list-style: none;
}
.run-info ul li {
  display: inline;
  margin-right: 6em;
  margin-left: -3em;
  text-transform: uppercase;
  font-size: small;
  color: #ec407a;
}
#content {
  max-width: 800px;
  margin-left: auto;
  margin-right: auto;
  padding-top: 2em;
  color: #263238;
  font-family: Helvetica, sans-serif;
}
p {
  margin: 0;
}
.command-table {
  background-color: #eceff1;
  padding: 0.3em;
  border-radius: 0.3em;
  margin-bottom: 4em;
}
.command-table .fail-ignored {
  background-color: #e65100cc;
  padding: 0.3em;
  border-radius: 0.3em;
}
.command-table .success {
  background-color: #1a237ecc;
  padding: 0.3em;
  border-radius: 0.3em;
}
.command-table .in-progress {
  background-color: black;
  padding: 0.3em;
  border-radius: 0.3em;
}
.command-table .non-started {
  background-color: #26323844;
  padding: 0.3em;
  border-radius: 0.3em;
}
.command-table .fail {
  background-color: #f44336cc;
  padding: 0.3em;
  border-radius: 0.3em;
}
.command-content {
  background-color: #eceff1;
  padding: 1em;
}
.command-header {
  display: flex;
  align-items: center;
  margin-bottom: 1em;
}
.command-id-link{
  flex-grow: 0;
}
.command-id-link a {
  font-size: xx-large;
  text-align: center;
  font-weight: bold;
  text-shadow: 1px  1px 0px #000000bb,
              -1px -1px 0px #ffffffbb;
  text-decoration: none;
  margin-left: 0.2em;
}
.command-id-link a:link {
  text-shadow: 1px  1px 0px #000000bb,
              -1px -1px 0px #ffffffbb;
}
.command-id-link a:visited {
  text-shadow: 1px  1px 0px #000000bb,
              -1px -1px 0px #ffffffbb;
}
.command-id-link a:visited:hover {
  color: #ec407a;
  background-color: #00000000;
  text-shadow: 1px  1px 0px #000000bb,
              -1px -1px 0px #ffffffbb,
               0px 0px 10px #fff,
               0px 0px 5px #ec407a,
               0px 0px 5px #ec407a;
}
.command-id-link a:hover {
  color: #ec407a;
  background-color: #00000000;
  text-shadow: 1px  1px 0px #000000bb,
              -1px -1px 0px #ffffffbb
               0px 0px 10px #fff,
               0px 0px 5px #ec407a,
               0px 0px 5px #ec407a;
}
.command-id-link a:active {
  color: #ffeb3b;
  background-color: #00000000;
  text-shadow: 1px  1px 0px #000000bb,
              -1px -1px 0px #ffffffbb,
                 0px 0px 10px #fff,
                 0px 0px 5px #ffeb3b,
                 0px 0px 5px #ffeb3b;
}
.command-id-link a:visited:active {
  color: #ffeb3b;
  background-color: #00000000;
  text-shadow: 1px  1px 0px #000000bb,
              -1px -1px 0px #ffffffbb,
                 0px 0px 10px #fff,
                 0px 0px 5px #ffeb3b,
                 0px 0px 5px #ffeb3b;
}
.command-description {
  color: #26323899;
  font-size: smaller;
  text-align: center;
  font-weight: bold;
  flex-grow: 1;
}

.command-invocation {
  padding: 1em;
  color: #fff;
  background-color: #005662;
  font-size: small;
  font-family: monospace;
  border-radius: 0.5em;
  text-shadow: 0 0 2px #ffccbccc;
}
.command-stats-table {
  margin-top: 0.2em;
  border-radius: 0.5em;
  padding: 0.2em;
  padding-top: 0.4em;
  display: flex;
  flex-direction: column;
  background-color: #eae3998a;
  color: #263238cc;
  margin-bottom: 0.2em;
}
.command-stats-row {
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
  margin-top: 0.4em;
  margin-bottom: 0.4em;
  font-size: small;
  text-transform: uppercase;
}
.command-stats-row p .value {
  color: #00251aaa;
  text-shadow: 0 0 2px #82ada944;
  font-weight: bold;
}
.literal-stdout {
  margin-top: 0.4em;
  margin-bottom: 0.2em;
  padding: 1em;
  border-radius: 0.5em;
  overflow-x: auto;
  overflow-y: auto;
}
.output-box {
  margin-top: 0.4em;
  margin-bottom: 0.2em;
  padding: 1em;
  color: #000a12;
  background-color: #cfd8dc;
  font-size: small;
  font-family: monospace;
  border-radius: 0.5em;
  max-height: 20em;
  overflow-x: auto;
  overflow-y: auto;
}
.output-box xmp{
  white-space: pre;
}
.stage-name p {
  margin-left: 0.5em;
  color: #546e7a;
  text-shadow:  -1px  0px 0px #fff
               ,-2px  0px 0px #ffffff99
               ,-1px  1px 0px #ffffff99
               ;
  font-size: 48pt;
  font-variant: small-caps;
  text-transform: lowercase;
  font-weight: bold;
  font-style: italic;
}
.stage-name .underline {
  width: 100%;
  height: 1.1em;
  background-color: #546e7a;
  margin-top: -1.6em;
}
.stage-name {
  margin-top: 6em;
  margin-bottom: 3em;
}

.outcome-table-info-box {
  margin-top: 0.4em;
  margin-bottom: 0.2em;
  padding: 1em;
  color: #000a12;
  background-color: #ffee58;
  border-radius: 0.5em;
}
.outcome-table-info-box .outcome{
  font-weight: bold;
}
#toc {
  margin-bottom: 6em;
  display: flex;
  flex-direction: column;
  flex-shrink: 1;
  margin-right: 3em;
}
.toc-title {
  text-align: center;
  font-variant: small-caps;
  text-transform: lowercase;
  margin-bottom: -1em;
}
.toc-section{
  margin-top: 1em;
  margin-bottom: 0.5em;
  text-transform: capitalize;
}
.toc-row {
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  align-items: center;
}
.toc-row-icon {
  padding-right: 8px;
}
.status-icon-cross {
  fill: #f44336;
    width: 16px;
    height: 16px;
}
.status-icon-tick {
  fill: #3f51b5;
  stroke: #ffffff77;
  stroke-width: 1px;
  width: 16px;
  height: 16px;
}

.status-icon-in-progress {
  fill: #333;
  width: 16px;
  height: 16px;
}

.status-icon-in-progress circle {
  stroke: #333;
  stroke-width: 2px;
  fill: none
}
a:link {
  color: #1e88e5;
}
a:visited {
  color: #1e88e5;
}
a:visited:hover {
  background-color: #1e88e5;
  color: #fff;
  text-decoration: none;
}
a:hover {
  background-color: #1e88e5;
  color: #fff;
  text-decoration: none;
}
a:active {
  background-color: #ec407a;
  color: #fff;
  text-decoration: none;
}
a:visited:active {
  background-color: #ec407a;
  color: #fff;
  text-decoration: none;
}

#contents-container {
  margin-top: 6em;
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  align-content: flex-start;
}

#depgraph {
  display: flex;
  flex-direction: column;
  align-items: center;
  align-content: flex-start;
  flex-shrink: 7;
}

#depgraph-title p {
  text-align: center;
  font-variant: small-caps;
  text-transform: lowercase;
}

#depgraph-preview img {
  max-width: 100%;
  max-height: 100%;
}

#depgraph-preview {
  padding: 1em;
}

.memory-box {
  display: flex;
  justify-content: space-between;
  padding: 0.5em;
  align-items: center;
}
.memory-preview {
  width: 75%;
}
.memory-preview svg {
  margin-left: -1em;
  margin-right: -2em;
}
.memory-table {
}
.memory-title {
  text-align: center;
  font-style: bold;
}
.memory-subtitle {
  text-transform: lowercase;
  font-variant: small-caps;
  text-align: center;
  opacity: 0.8;
}
.memory-pad {
  width: 1.5em;
}
.memory-value {
  text-align: center;
  margin-right: 2em;
}
.rss {
  color: #ab47bc;
}
.vsz {
  color: #4caf50;
}


@media (prefers-color-scheme: dark){
  a:link {
    color: #29b6f6;
  }
  a:visited {
    color: #29b6f6;
  }
  a:visited:hover {
    color: #263238;
    background-color: #29b6f6;
  }
  a:hover {
    color: #263238;
    background-color: #29b6f6;
  }
  a:active {
    color: #263238;
    background-color: #ec407a;
  }
  a:visited:active {
    color: #263238;
    background-color: #ec407a;
  }
  .command-id-link a:link {
    text-shadow: 1px  1px 0px #000000bb,
                -1px -1px 0px #ffffffbb;
  }
  .command-id-link a:visited {
    text-shadow: 1px  1px 0px #000000bb,
                -1px -1px 0px #ffffffbb;
  }
  .command-id-link a:visited:hover {
    color: #ec407a;
    background-color: #00000000;
    text-shadow: 1px  1px 0px #000000bb,
                -1px -1px 0px #ffffffbb.
                 0px 0px 10px #fff,
                 0px 0px 5px #ec407a,
                 0px 0px 5px #ec407a;
  }
  .command-id-link a:hover {
    color: #ec407a;
    background-color: #00000000;
    text-shadow: 1px  1px 0px #000000bb,
                -1px -1px 0px #ffffffbb,
                 0px 0px 10px #fff,
                 0px 0px 5px #ec407a,
                 0px 0px 5px #ec407a;
  }
  .command-id-link a:active {
    color: #ffeb3b;
    background-color: #00000000;
    text-shadow: 1px  1px 0px #000000bb,
                -1px -1px 0px #ffffffbb,
                 0px 0px 10px #fff,
                 0px 0px 5px #ffeb3b,
                 0px 0px 5px #ffeb3b;
  }
  .command-id-link a:visited:active {
    color: #ffeb3b;
    background-color: #00000000;
    text-shadow: 1px  1px 0px #000000bb,
                -1px -1px 0px #ffffffbb,
                 0px 0px 10px #fff,
                 0px 0px 5px #ffeb3b,
                 0px 0px 5px #ffeb3b;
  }
  body {
    color: #babdbe;
    background-color: #263238;
  }
  .outcome-table-info-box p{
    color: #000a12;
  }
  p {
    color: #babdbe;
  }
  #subtitle {
    color: #fff;
  }
  .command-table {
    background-color: #37474f;
  }
  .command-content {
    background-color: #37474f;
  }
  .command-invocation p {
    color: #fff;
  }
  .command-stats-table {
    background-color: #eae399d0;
    color: #263238;
  }
  .command-stats-table p {
    color: #263238;
  }
  .output-box {
    color: #cfd8dccc;
    background-color: #102027;
  }
  .stage-name p {
    text-shadow: -1px  0px 0px #263238,
                 -2px  0px 0px #26323899,
                 -1px  1px 0px #26323899;
  }
  .status-icon-in-progress { fill: #ddd; }
  .status-icon-in-progress circle { stroke: #ddd; }

  /* SVG graphs */
  g text {
    fill: #eceff1;
    stroke: #fff0;
  }
}
</style>
</head>
<body>
<div id="content">

  <div id="title">
    <h1>
      Book Runner pipeline &lsquo;tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs&rsquo;
    </h1>
    <p id="subtitle">
      Litani CI Dashboard
    </p>
  </div><!-- id="title" -->

  <div class=run-info>
    <ul>
      <li>Pipeline status: success</li>
      <li>2022-06-28T17:18:50Z</li>
      <li>8c150595-9290-4fd6-bf99-9894dbe0c928</li>
    </ul>
  </div><!-- class=run-info -->

  <div id="contents-container">

    <div id="toc">
      <div class="toc-title">
        <p>Summary of jobs for this pipeline</p>
      </div><!-- class="toc-title" -->

      
        <div class="toc-section">
          <p>check</p>
        </div><!-- class="toc-section" -->
        
          <div class="toc-row">

            <div class="toc-row-icon">
              
                <svg height="16px" width="16px" class="status-icon-tick">
                  <polygon points="1,11 6,15 15,3 13,1 6,12 3,8"></polygon>
                </svg>
              
            </div><!-- class="toc-row-icon" -->

            <div class="toc-row-job-name">
              <p>
              <a href="#job-8b7e20c6-681b-497e-93b3-57edc2771bf2">
              
              Is this valid Rust code?
              
              </a>
              </p>
            </div><!-- class="toc-row-job-name" -->

          </div><!-- class="toc-row" -->

        
      
        <div class="toc-section">
          <p>codegen</p>
        </div><!-- class="toc-section" -->
        
          <div class="toc-row">

            <div class="toc-row-icon">
              
                <svg height="16px" width="16px" class="status-icon-tick">
                  <polygon points="1,11 6,15 15,3 13,1 6,12 3,8"></polygon>
                </svg>
              
            </div><!-- class="toc-row-icon" -->

            <div class="toc-row-job-name">
              <p>
              <a href="#job-7373cc4c-ed4f-46d6-829e-1be89992ffd2">
              
              Does Kani support all the Rust features used in it?
              
              </a>
              </p>
            </div><!-- class="toc-row-job-name" -->

          </div><!-- class="toc-row" -->

        
      
        <div class="toc-section">
          <p>verification</p>
        </div><!-- class="toc-section" -->
        
          <div class="toc-row">

            <div class="toc-row-icon">
              
                <svg height="16px" width="16px" class="status-icon-tick">
                  <polygon points="1,11 6,15 15,3 13,1 6,12 3,8"></polygon>
                </svg>
              
            </div><!-- class="toc-row-icon" -->

            <div class="toc-row-job-name">
              <p>
              <a href="#job-c04cbec4-8eca-454a-b361-f482143d6537">
              
              Ca Kani reason about it?
              
              </a>
              </p>
            </div><!-- class="toc-row-job-name" -->

          </div><!-- class="toc-row" -->

        
      
    </div><!-- id="toc" -->


    

  </div><!-- id="contents-container" -->


  
  <div class="stage">
    <div class="stage-name">
      <p>check</p>
      <div class="underline"></div>
    </div><!-- class="stage-name" -->

    
    <div class="command-table" id="job-8b7e20c6-681b-497e-93b3-57edc2771bf2">
      
      <div class="success">
      
        <div class="command-content">
          <div class="command-header">
            <div class="command-id-link">
              <p>
                <a href="#job-8b7e20c6-681b-497e-93b3-57edc2771bf2">#</a>
              </p>
            </div><!-- class="command-id-link" -->
            <div class="command-description">
              
              <p>
                Is this valid Rust code?
              </p>
              
            </div><!-- class="command-description" -->
          </div><!-- class="command-header" -->
          <div class="command-invocation">
            <p>
            "kani-compiler" "--edition" "2015" "-Z" "no-codegen" "tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs"
            </p>
          </div><!-- class="command-invocation" -->

          
          <div class="command-stats-table">
            
            <div class="command-stats-row">
              <p>Command return code:
              <span class="value">
                0
              </span></p>
              <p>OK returns:
              <span class="value">
                ['0']
              </span></p>
              <p>Ignored returns:
              <span class="value">
                None
              </span></p>
            </div><!-- class="command-stats-row" -->
            <div class="command-stats-row">
              <p>Timeout:
              <span class="value">
                5
              </span></p>
              <p>Timeout ok:
              <span class="value">
                False
              </span></p>
              <p>Timeout ignored:
              <span class="value">
                False
              </span></p>
              <p>Timeout reached:
              <span class="value">
                False
              </span></p>
            </div><!-- class="command-stats-row" -->
            <div class="command-stats-row">
              <p>Start time:
              <span class="value">
                2022-06-28T17:24:39Z
              </span></p>
              <p>Duration:
              <span class="value">
                00s
              </span></p>
              <p>End time:
              <span class="value">
                2022-06-28T17:24:39Z
              </span></p>
            </div><!-- class="command-stats-row" -->
            <div class="command-stats-row">
              <p>Command successful:
              <span class="value">
              
              yes
              
              </span></p>
            </div><!-- class="command-stats-row" -->
             
          </div><!-- class="command-stats-table" -->
          

          


          


          

          
          <div class="output-box"><xmp>stderr:
            
warning: this method call resolves to `<&[T; N] as IntoIterator>::into_iter` (due to backwards compatibility), but will resolve to <[T; N] as IntoIterator>::into_iter in Rust 2021
--> tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:19:40
|
19 |     println!("2 in array2: {}", array2.into_iter().any(|&x| x == 2));
|                                        ^^^^^^^^^
|
= note: `#[warn(array_into_iter)]` on by default
= warning: this changes meaning in Rust 2021
= note: for more information, see <https://doc.rust-lang.org/nightly/edition-guide/rust-2021/IntoIterator-for-arrays.html>
help: use `.iter()` instead of `.into_iter()` to avoid ambiguity
|
19 |     println!("2 in array2: {}", array2.iter().any(|&x| x == 2));
|                                        ~~~~
help: or use `IntoIterator::into_iter(..)` instead of `.into_iter()` to explicitly iterate by value
|
19 |     println!("2 in array2: {}", IntoIterator::into_iter(array2).any(|&x| x == 2));
|                                 ++++++++++++++++++++++++      ~

warning: 1 warning emitted

            </xmp>
          </div><!-- class="output-box" -->
          

        </div><!-- class="command-content" -->
      </div><!-- class="fail" -->
    </div><!-- id="command-table" -->
    

  </div><!-- class="stage" -->
  
  <div class="stage">
    <div class="stage-name">
      <p>codegen</p>
      <div class="underline"></div>
    </div><!-- class="stage-name" -->

    
    <div class="command-table" id="job-7373cc4c-ed4f-46d6-829e-1be89992ffd2">
      
      <div class="success">
      
        <div class="command-content">
          <div class="command-header">
            <div class="command-id-link">
              <p>
                <a href="#job-7373cc4c-ed4f-46d6-829e-1be89992ffd2">#</a>
              </p>
            </div><!-- class="command-id-link" -->
            <div class="command-description">
              
              <p>
                Does Kani support all the Rust features used in it?
              </p>
              
            </div><!-- class="command-description" -->
          </div><!-- class="command-header" -->
          <div class="command-invocation">
            <p>
            "kani-compiler" "--edition" "2015" "--out-dir" "build/tmp" "tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs"
            </p>
          </div><!-- class="command-invocation" -->

          
          <div class="command-stats-table">
            
            <div class="command-stats-row">
              <p>Command return code:
              <span class="value">
                0
              </span></p>
              <p>OK returns:
              <span class="value">
                ['0']
              </span></p>
              <p>Ignored returns:
              <span class="value">
                None
              </span></p>
            </div><!-- class="command-stats-row" -->
            <div class="command-stats-row">
              <p>Timeout:
              <span class="value">
                10
              </span></p>
              <p>Timeout ok:
              <span class="value">
                False
              </span></p>
              <p>Timeout ignored:
              <span class="value">
                False
              </span></p>
              <p>Timeout reached:
              <span class="value">
                False
              </span></p>
            </div><!-- class="command-stats-row" -->
            <div class="command-stats-row">
              <p>Start time:
              <span class="value">
                2022-06-28T17:24:39Z
              </span></p>
              <p>Duration:
              <span class="value">
                01s
              </span></p>
              <p>End time:
              <span class="value">
                2022-06-28T17:24:40Z
              </span></p>
            </div><!-- class="command-stats-row" -->
            <div class="command-stats-row">
              <p>Command successful:
              <span class="value">
              
              yes
              
              </span></p>
            </div><!-- class="command-stats-row" -->
             
          </div><!-- class="command-stats-table" -->
          

          


          


          

          
          <div class="output-box"><xmp>stderr:
            
warning: this method call resolves to `<&[T; N] as IntoIterator>::into_iter` (due to backwards compatibility), but will resolve to <[T; N] as IntoIterator>::into_iter in Rust 2021
--> tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:19:40
|
19 |     println!("2 in array2: {}", array2.into_iter().any(|&x| x == 2));
|                                        ^^^^^^^^^
|
= note: `#[warn(array_into_iter)]` on by default
= warning: this changes meaning in Rust 2021
= note: for more information, see <https://doc.rust-lang.org/nightly/edition-guide/rust-2021/IntoIterator-for-arrays.html>
help: use `.iter()` instead of `.into_iter()` to avoid ambiguity
|
19 |     println!("2 in array2: {}", array2.iter().any(|&x| x == 2));
|                                        ~~~~
help: or use `IntoIterator::into_iter(..)` instead of `.into_iter()` to explicitly iterate by value
|
19 |     println!("2 in array2: {}", IntoIterator::into_iter(array2).any(|&x| x == 2));
|                                 ++++++++++++++++++++++++      ~

warning: 1 warning emitted

            </xmp>
          </div><!-- class="output-box" -->
          

        </div><!-- class="command-content" -->
      </div><!-- class="fail" -->
    </div><!-- id="command-table" -->
    

  </div><!-- class="stage" -->
  
  <div class="stage">
    <div class="stage-name">
      <p>verification</p>
      <div class="underline"></div>
    </div><!-- class="stage-name" -->

    
    <div class="command-table" id="job-c04cbec4-8eca-454a-b361-f482143d6537">
      
      <div class="success">
      
        <div class="command-content">
          <div class="command-header">
            <div class="command-id-link">
              <p>
                <a href="#job-c04cbec4-8eca-454a-b361-f482143d6537">#</a>
              </p>
            </div><!-- class="command-id-link" -->
            <div class="command-description">
              
              <p>
                Ca Kani reason about it?
              </p>
              
            </div><!-- class="command-description" -->
          </div><!-- class="command-header" -->
          <div class="command-invocation">
            <p>
            RUSTFLAGS="--edition 2015" "kani" "tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs" "--enable-unstable" "--function" "main" "--enable-unstable" "--cbmc-args" "--unwind" "4"
            </p>
          </div><!-- class="command-invocation" -->

          
          <div class="command-stats-table">
            
            <div class="command-stats-row">
              <p>Command return code:
              <span class="value">
                0
              </span></p>
              <p>OK returns:
              <span class="value">
                ['0']
              </span></p>
              <p>Ignored returns:
              <span class="value">
                None
              </span></p>
            </div><!-- class="command-stats-row" -->
            <div class="command-stats-row">
              <p>Timeout:
              <span class="value">
                60
              </span></p>
              <p>Timeout ok:
              <span class="value">
                False
              </span></p>
              <p>Timeout ignored:
              <span class="value">
                False
              </span></p>
              <p>Timeout reached:
              <span class="value">
                False
              </span></p>
            </div><!-- class="command-stats-row" -->
            <div class="command-stats-row">
              <p>Start time:
              <span class="value">
                2022-06-28T17:25:42Z
              </span></p>
              <p>Duration:
              <span class="value">
                05s
              </span></p>
              <p>End time:
              <span class="value">
                2022-06-28T17:25:47Z
              </span></p>
            </div><!-- class="command-stats-row" -->
            <div class="command-stats-row">
              <p>Command successful:
              <span class="value">
              
              yes
              
              </span></p>
            </div><!-- class="command-stats-row" -->
             
          </div><!-- class="command-stats-table" -->
          

          


          


          
            
            <div class="output-box"><xmp>stdout:
              
Checking harness main...
CBMC 5.60.0 (cbmc-5.60.0)
CBMC version 5.60.0 (cbmc-5.60.0) 64-bit x86_64 linux
Reading GOTO program from file tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.out.for-main
Generating GOTO Program
Adding CPROVER library (x86_64)
Removal of function pointers and virtual functions
Generic Property Instrumentation
Running with 16 object bits, 48 offset bits (user-specified)
Starting Bounded Model Checking
Unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224main0EB1F_.0 iteration 1 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224main0EB1F_.0 iteration 2 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224main0EB1F_.0 iteration 3 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Not unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224main0EB1F_.0 iteration 4 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs line 109 column 15 function <std::ops::ControlFlow<B, C> as std::ops::Try>::branch thread 0
Unwinding loop _RINvYINtNtNtCsc53D9JaGOED_5alloc3vec9into_iter8IntoIterlENtNtNtNtCslOTGKtp7Iyc_4core4iter6traits8iterator8Iterator8try_folduNCINvNvBT_3any5checklNCNvCsclqJl7O4wFo_2_224mains_0E0INtNtNtB11_3ops12control_flow11ControlFlowuEEB2n_.0 iteration 1 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/iterator.rs line 2189 column 9 function std::iter::Iterator::try_fold thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs line 109 column 15 function <std::ops::ControlFlow<B, C> as std::ops::Try>::branch thread 0
Unwinding loop _RINvYINtNtNtCsc53D9JaGOED_5alloc3vec9into_iter8IntoIterlENtNtNtNtCslOTGKtp7Iyc_4core4iter6traits8iterator8Iterator8try_folduNCINvNvBT_3any5checklNCNvCsclqJl7O4wFo_2_224mains_0E0INtNtNtB11_3ops12control_flow11ControlFlowuEEB2n_.0 iteration 2 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/iterator.rs line 2189 column 9 function std::iter::Iterator::try_fold thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs line 109 column 15 function <std::ops::ControlFlow<B, C> as std::ops::Try>::branch thread 0
Unwinding loop _RINvYINtNtNtCsc53D9JaGOED_5alloc3vec9into_iter8IntoIterlENtNtNtNtCslOTGKtp7Iyc_4core4iter6traits8iterator8Iterator8try_folduNCINvNvBT_3any5checklNCNvCsclqJl7O4wFo_2_224mains_0E0INtNtNtB11_3ops12control_flow11ControlFlowuEEB2n_.0 iteration 3 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/iterator.rs line 2189 column 9 function std::iter::Iterator::try_fold thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs line 109 column 15 function <std::ops::ControlFlow<B, C> as std::ops::Try>::branch thread 0
Not unwinding loop _RINvYINtNtNtCsc53D9JaGOED_5alloc3vec9into_iter8IntoIterlENtNtNtNtCslOTGKtp7Iyc_4core4iter6traits8iterator8Iterator8try_folduNCINvNvBT_3any5checklNCNvCsclqJl7O4wFo_2_224mains_0E0INtNtNtB11_3ops12control_flow11ControlFlowuEEB2n_.0 iteration 4 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/iterator.rs line 2189 column 9 function std::iter::Iterator::try_fold thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/iterator.rs line 2190 column 21 function std::iter::Iterator::try_fold thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs line 82 column 30 function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq thread 0
aborting path on assume(false) at  thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/panicking.rs line 181 column 5 function core::panicking::assert_failed thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/macros/mod.rs line 45 column 21 function std::iter::ExactSizeIterator::len thread 0
Unwinding loop _RNvMs_NtCsc53D9JaGOED_5alloc7raw_vecINtB4_6RawVeclE14current_memoryCsclqJl7O4wFo_2_22.1 iteration 1  thread 0
Unwinding loop _RINvMNtCslOTGKtp7Iyc_4core6optionINtB3_6OptionjE5ok_orNtNtNtB5_5alloc6layout11LayoutErrorECsclqJl7O4wFo_2_22.1 iteration 1  thread 0
Unwinding loop _RINvMNtCslOTGKtp7Iyc_4core6optionINtB3_6OptionjE5ok_orNtNtNtB5_5alloc6layout11LayoutErrorECsclqJl7O4wFo_2_22.0 iteration 1  thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/hint.rs line 102 column 14 function std::hint::unreachable_unchecked thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs line 1516 column 32 function std::result::Result::<T, E>::unwrap_unchecked thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs line 1513 column 15 function std::result::Result::<T, E>::unwrap_unchecked thread 0
Unwinding loop _RNvMNtCslOTGKtp7Iyc_4core6resultINtB2_6ResultNtNtNtB4_5alloc6layout6LayoutNtBJ_11LayoutErrorE16unwrap_uncheckedCsclqJl7O4wFo_2_22.0 iteration 1  thread 0
Unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224mains0_0EB1F_.0 iteration 1 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224mains0_0EB1F_.0 iteration 2 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224mains0_0EB1F_.0 iteration 3 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Not unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224mains0_0EB1F_.0 iteration 4 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224mains1_0EB1F_.0 iteration 1 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224mains1_0EB1F_.0 iteration 2 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224mains1_0EB1F_.0 iteration 3 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Not unwinding loop _RINvXs2L_NtNtCslOTGKtp7Iyc_4core5slice4iterINtB7_4IterlENtNtNtNtBb_4iter6traits8iterator8Iterator3anyNCNvCsclqJl7O4wFo_2_224mains1_0EB1F_.0 iteration 4 file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs line 241 column 17 function <std::slice::Iter<'a, T> as std::iter::Iterator>::any thread 0
Unwinding loop _RNvMs_NtCsc53D9JaGOED_5alloc7raw_vecINtB4_6RawVeclE14current_memoryCsclqJl7O4wFo_2_22.1 iteration 1  thread 0
Unwinding loop _RINvMNtCslOTGKtp7Iyc_4core6optionINtB3_6OptionjE5ok_orNtNtNtB5_5alloc6layout11LayoutErrorECsclqJl7O4wFo_2_22.1 iteration 1  thread 0
Unwinding loop _RINvMNtCslOTGKtp7Iyc_4core6optionINtB3_6OptionjE5ok_orNtNtNtB5_5alloc6layout11LayoutErrorECsclqJl7O4wFo_2_22.0 iteration 1  thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/hint.rs line 102 column 14 function std::hint::unreachable_unchecked thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs line 1516 column 32 function std::result::Result::<T, E>::unwrap_unchecked thread 0
aborting path on assume(false) at file /home/runner/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs line 1513 column 15 function std::result::Result::<T, E>::unwrap_unchecked thread 0
Unwinding loop _RNvMNtCslOTGKtp7Iyc_4core6resultINtB2_6ResultNtNtNtB4_5alloc6layout6LayoutNtBJ_11LayoutErrorE16unwrap_uncheckedCsclqJl7O4wFo_2_22.0 iteration 1  thread 0
Unwinding loop _RINvNtCslOTGKtp7Iyc_4core3ptr13drop_in_placeINtNtCsc53D9JaGOED_5alloc3vec3VeclEECsclqJl7O4wFo_2_22.0 iteration 1  thread 0
Runtime Symex: 0.483892s
size of program expression: 11241 steps
slicing removed 7209 assignments
Generated 1128 VCC(s), 404 remaining after simplification
Runtime Postprocess Equation: 0.00571611s
Passing problem to propositional reduction
converting SSA
Runtime Convert SSA: 0.0496359s
Running propositional reduction
Post-processing
Runtime Post-process: 0.0300018s
Solving with MiniSAT 2.2.1 with simplifier
38277 variables, 65443 clauses
SAT checker: instance is UNSATISFIABLE
Runtime Solver: 0.121648s
Runtime decision procedure: 0.172312s
VERIFICATION SUCCESSFUL

RESULTS:
Check 1: __rust_dealloc.assertion.1
- Status: SUCCESS
- Description: "Alignment is power of two"
- Location: library/kani/kani_lib.c:74 in function __rust_dealloc

Check 2: __rust_dealloc.assertion.2
- Status: SUCCESS
- Description: "rust_dealloc must be called on an object whose allocated size matches its layout"
- Location: library/kani/kani_lib.c:76 in function __rust_dealloc

Check 3: std::ptr::const_ptr::<impl *const T>::sub_ptr.assertion.1
- Status: SUCCESS
- Description: "assertion failed: 0 < pointee_size && pointee_size <= isize::MAX as usize"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/const_ptr.rs:743:9 in function std::ptr::const_ptr::<impl *const T>::sub_ptr

Check 4: std::ptr::const_ptr::<impl *const T>::sub_ptr.arithmetic_overflow.1
- Status: SUCCESS
- Description: "attempt to compute offset in bytes which would overflow an `isize`"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/const_ptr.rs:745:18 in function std::ptr::const_ptr::<impl *const T>::sub_ptr

Check 5: std::ptr::const_ptr::<impl *const T>::sub_ptr.safety_check.1
- Status: SUCCESS
- Description: "attempt to compute unsigned offset with negative distance"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/const_ptr.rs:745:18 in function std::ptr::const_ptr::<impl *const T>::sub_ptr

Check 6: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.assume.1
- Status: SUCCESS
- Description: "assumption failed"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:142:21 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 7: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.assume.2
- Status: SUCCESS
- Description: "assumption failed"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:144:25 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 8: <std::slice::Iter<'a, T> as std::iter::Iterator>::any.unsupported_construct.1
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:236:13 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::any

Check 9: <std::slice::Iter<'a, T> as std::iter::Iterator>::any.unsupported_construct.2
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:236:13 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::any

Check 10: <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter.arithmetic_overflow.1
- Status: SUCCESS
- Description: "arith_offset: attempt to compute number in bytes which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2648:17 in function <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter

Check 11: <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter.arithmetic_overflow.2
- Status: SUCCESS
- Description: "attempt to compute offset which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2648:17 in function <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter

Check 12: malloc.assertion.1
- Status: SUCCESS
- Description: "max allocation size exceeded"
- Location: <builtin-library-malloc>:28 in function malloc

Check 13: malloc.assertion.2
- Status: SUCCESS
- Description: "max allocation may fail"
- Location: <builtin-library-malloc>:33 in function malloc

Check 14: std::result::Result::<T, E>::unwrap_unchecked.unreachable.1
- Status: SUCCESS
- Description: "unreachable code"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:1513:15 in function std::result::Result::<T, E>::unwrap_unchecked

Check 15: std::result::Result::<T, E>::unwrap_unchecked.unsupported_construct.1
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:1511:5 in function std::result::Result::<T, E>::unwrap_unchecked

Check 16: std::intrinsics::write_bytes.safety_check.1
- Status: SUCCESS
- Description: "`dst` must be properly aligned"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/intrinsics.rs:2303:9 in function std::intrinsics::write_bytes

Check 17: std::intrinsics::write_bytes.arithmetic_overflow.1
- Status: SUCCESS
- Description: "write_bytes: attempt to compute number in bytes which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/intrinsics.rs:2303:9 in function std::intrinsics::write_bytes

Check 18: std::ptr::mut_ptr::<impl *mut T>::offset.arithmetic_overflow.1
- Status: SUCCESS
- Description: "offset: attempt to compute number in bytes which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mut_ptr.rs:469:18 in function std::ptr::mut_ptr::<impl *mut T>::offset

Check 19: std::ptr::mut_ptr::<impl *mut T>::offset.arithmetic_overflow.2
- Status: SUCCESS
- Description: "attempt to compute offset which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mut_ptr.rs:469:18 in function std::ptr::mut_ptr::<impl *mut T>::offset

Check 20: std::vec::Vec::<T, A>::as_mut_ptr.assume.1
- Status: SUCCESS
- Description: "assumption failed"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:1180:13 in function std::vec::Vec::<T, A>::as_mut_ptr

Check 21: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.unsupported_construct.1
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:319:13 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 22: <std::slice::Iter<'a, T> as std::iter::Iterator>::any.unsupported_construct.3
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:236:13 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::any

Check 23: <std::ops::ControlFlow<B, C> as std::ops::Try>::branch.unreachable.1
- Status: SUCCESS
- Description: "unreachable code"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:109:15 in function <std::ops::ControlFlow<B, C> as std::ops::Try>::branch

Check 24: std::option::Option::<T>::ok_or.unreachable.1
- Status: SUCCESS
- Description: "unreachable code"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:1055:15 in function std::option::Option::<T>::ok_or

Check 25: alloc::raw_vec::RawVec::<T, A>::from_raw_parts_in.unsupported_construct.1
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:215:5 in function alloc::raw_vec::RawVec::<T, A>::from_raw_parts_in

Check 26: <std::result::Result<T, E> as std::ops::Try>::branch.unreachable.1
- Status: SUCCESS
- Description: "unreachable code"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:2109:15 in function <std::result::Result<T, E> as std::ops::Try>::branch

Check 27: std::iter::Iterator::try_fold.unreachable.1
- Status: SUCCESS
- Description: "unreachable code"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/iterator.rs:2190:21 in function std::iter::Iterator::try_fold

Check 28: std::iter::Iterator::try_fold.unsupported_construct.1
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/iterator.rs:2182:5 in function std::iter::Iterator::try_fold

Check 29: std::alloc::Layout::array.unreachable.1
- Status: SUCCESS
- Description: "unreachable code"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:407:26 in function std::alloc::Layout::array

Check 30: std::ptr::const_ptr::<impl *const T>::offset.arithmetic_overflow.1
- Status: SUCCESS
- Description: "offset: attempt to compute number in bytes which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/const_ptr.rs:457:18 in function std::ptr::const_ptr::<impl *const T>::offset

Check 31: std::ptr::const_ptr::<impl *const T>::offset.arithmetic_overflow.2
- Status: SUCCESS
- Description: "attempt to compute offset which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/const_ptr.rs:457:18 in function std::ptr::const_ptr::<impl *const T>::offset

Check 32: std::vec::Vec::<T, A>::as_ptr.assume.1
- Status: SUCCESS
- Description: "assumption failed"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:1143:13 in function std::vec::Vec::<T, A>::as_ptr

Check 33: _ZN4core9panicking19assert_failed_inner17hcaaeca3596f28e63E.1
- Status: SUCCESS
- Description: "assertion"
- Location: Unknown File in function _ZN4core9panicking19assert_failed_inner17hcaaeca3596f28e63E

Check 34: std::boxed::Box::<T, A>::into_unique.unsupported_construct.1
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1136:5 in function std::boxed::Box::<T, A>::into_unique

Check 35: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.arithmetic_overflow.1
- Status: SUCCESS
- Description: "arith_offset: attempt to compute number in bytes which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:33 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 36: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.arithmetic_overflow.2
- Status: SUCCESS
- Description: "attempt to compute offset which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:33 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 37: std::slice::Iter::<'a, T>::new.assume.1
- Status: SUCCESS
- Description: "assumption failed"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter.rs:92:13 in function std::slice::Iter::<'a, T>::new

Check 38: alloc::alloc::box_free.unsupported_construct.1
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/alloc.rs:334:1 in function alloc::alloc::box_free

Check 39: std::hint::unreachable_unchecked.unreachable.1
- Status: SUCCESS
- Description: "unreachable code"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/hint.rs:102:14 in function std::hint::unreachable_unchecked

Check 40: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.unreachable.1
- Status: SUCCESS
- Description: "unreachable code"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 41: std::ptr::const_ptr::<impl *const T>::wrapping_offset.arithmetic_overflow.1
- Status: SUCCESS
- Description: "arith_offset: attempt to compute number in bytes which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/const_ptr.rs:539:18 in function std::ptr::const_ptr::<impl *const T>::wrapping_offset

Check 42: std::ptr::const_ptr::<impl *const T>::wrapping_offset.arithmetic_overflow.2
- Status: SUCCESS
- Description: "attempt to compute offset which would overflow"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/const_ptr.rs:539:18 in function std::ptr::const_ptr::<impl *const T>::wrapping_offset

Check 43: <std::vec::IntoIter<T, A> as std::ops::Drop>::drop.unsupported_construct.1
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:315:5 in function <std::vec::IntoIter<T, A> as std::ops::Drop>::drop

Check 44: std::slice::hack::into_vec.unsupported_construct.1
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/slice.rs:167:5 in function std::slice::hack::into_vec

Check 45: std::boxed::Box::<T, A>::into_raw_with_allocator.unsupported_construct.1
- Status: SUCCESS
- Description: "resume instruction"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1123:5 in function std::boxed::Box::<T, A>::into_raw_with_allocator

Check 46: main.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 47: main.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 48: main.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 49: main.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 50: main.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 51: main.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 52: main.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 53: main.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 54: main.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 55: main.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 56: main.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 57: main.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/macros.rs:52:37 in function main

Check 58: std::vec::IntoIter::<T, A>::as_raw_mut_slice.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:93:39 in function std::vec::IntoIter::<T, A>::as_raw_mut_slice

Check 59: std::vec::IntoIter::<T, A>::as_raw_mut_slice.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:93:39 in function std::vec::IntoIter::<T, A>::as_raw_mut_slice

Check 60: std::vec::IntoIter::<T, A>::as_raw_mut_slice.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:93:39 in function std::vec::IntoIter::<T, A>::as_raw_mut_slice

Check 61: std::vec::IntoIter::<T, A>::as_raw_mut_slice.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:93:39 in function std::vec::IntoIter::<T, A>::as_raw_mut_slice

Check 62: std::vec::IntoIter::<T, A>::as_raw_mut_slice.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:93:39 in function std::vec::IntoIter::<T, A>::as_raw_mut_slice

Check 63: std::vec::IntoIter::<T, A>::as_raw_mut_slice.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:93:39 in function std::vec::IntoIter::<T, A>::as_raw_mut_slice

Check 64: <std::slice::Iter<'a, T> as std::iter::Iterator>::any.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:241:27 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::any

Check 65: main::{closure#0}.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:9:54 in function main::{closure#0}

Check 66: main::{closure#0}.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:9:54 in function main::{closure#0}

Check 67: main::{closure#0}.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:9:54 in function main::{closure#0}

Check 68: main::{closure#0}.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:9:54 in function main::{closure#0}

Check 69: main::{closure#0}.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:9:54 in function main::{closure#0}

Check 70: main::{closure#0}.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:9:54 in function main::{closure#0}

Check 71: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:322:52 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 72: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:322:52 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 73: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:322:52 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 74: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:322:52 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 75: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:322:52 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 76: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:322:52 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 77: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 78: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 79: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 80: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 81: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 82: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 83: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.13
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 84: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.14
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 85: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.15
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 86: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.16
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 87: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.17
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 88: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.18
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:55 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 89: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.19
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 90: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.20
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 91: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.21
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 92: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.22
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 93: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.23
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 94: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.24
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 95: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.25
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 96: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.26
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 97: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.27
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 98: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.28
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 99: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.29
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 100: <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop.pointer_dereference.30
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:324:76 in function <<std::vec::IntoIter<T, A> as std::ops::Drop>::drop::DropGuard<'_, T, A> as std::ops::Drop>::drop

Check 101: alloc::raw_vec::RawVec::<T, A>::capacity.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:232:59 in function alloc::raw_vec::RawVec::<T, A>::capacity

Check 102: alloc::raw_vec::RawVec::<T, A>::capacity.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:232:59 in function alloc::raw_vec::RawVec::<T, A>::capacity

Check 103: alloc::raw_vec::RawVec::<T, A>::capacity.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:232:59 in function alloc::raw_vec::RawVec::<T, A>::capacity

Check 104: alloc::raw_vec::RawVec::<T, A>::capacity.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:232:59 in function alloc::raw_vec::RawVec::<T, A>::capacity

Check 105: alloc::raw_vec::RawVec::<T, A>::capacity.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:232:59 in function alloc::raw_vec::RawVec::<T, A>::capacity

Check 106: alloc::raw_vec::RawVec::<T, A>::capacity.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:232:59 in function alloc::raw_vec::RawVec::<T, A>::capacity

Check 107: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 108: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 109: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 110: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 111: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 112: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 113: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:42 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 114: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:42 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 115: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:42 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 116: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:42 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 117: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:42 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 118: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:170:42 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 119: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.13
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:22 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 120: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.14
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:22 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 121: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.15
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:22 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 122: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.16
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:22 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 123: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.17
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:22 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 124: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.18
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:22 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 125: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.19
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:39 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 126: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.20
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:39 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 127: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.21
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:39 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 128: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.22
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:39 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 129: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.23
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:39 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 130: <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint.pointer_dereference.24
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:172:39 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::size_hint

Check 131: std::intrinsics::write_bytes.precondition_instance.1
- Status: SUCCESS
- Description: "memset destination region writeable"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/intrinsics.rs:2303:9 in function std::intrinsics::write_bytes

Check 132: pointer.1
- Status: SUCCESS
- Description: "same object violation"

Check 133: std::boxed::Box::<T, A>::leak.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1200:24 in function std::boxed::Box::<T, A>::leak

Check 134: std::boxed::Box::<T, A>::leak.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1200:24 in function std::boxed::Box::<T, A>::leak

Check 135: std::boxed::Box::<T, A>::leak.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1200:24 in function std::boxed::Box::<T, A>::leak

Check 136: std::boxed::Box::<T, A>::leak.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1200:24 in function std::boxed::Box::<T, A>::leak

Check 137: std::boxed::Box::<T, A>::leak.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1200:24 in function std::boxed::Box::<T, A>::leak

Check 138: std::boxed::Box::<T, A>::leak.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1200:24 in function std::boxed::Box::<T, A>::leak

Check 139: __rust_dealloc.precondition_instance.1
- Status: SUCCESS
- Description: "free argument must be NULL or valid pointer"
- Location: library/kani/kani_lib.c:78 in function __rust_dealloc

Check 140: __rust_dealloc.precondition_instance.2
- Status: SUCCESS
- Description: "free argument must be dynamic object"
- Location: library/kani/kani_lib.c:78 in function __rust_dealloc

Check 141: __rust_dealloc.precondition_instance.3
- Status: SUCCESS
- Description: "free argument has offset zero"
- Location: library/kani/kani_lib.c:78 in function __rust_dealloc

Check 142: __rust_dealloc.precondition_instance.4
- Status: SUCCESS
- Description: "double free"
- Location: library/kani/kani_lib.c:78 in function __rust_dealloc

Check 143: __rust_dealloc.precondition_instance.5
- Status: SUCCESS
- Description: "free called for new[] object"
- Location: library/kani/kani_lib.c:78 in function __rust_dealloc

Check 144: __rust_dealloc.precondition_instance.6
- Status: SUCCESS
- Description: "free called for stack-allocated object"
- Location: library/kani/kani_lib.c:78 in function __rust_dealloc

Check 145: <alloc::raw_vec::RawVec<T, A> as std::ops::Drop>::drop.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:477:16 in function <alloc::raw_vec::RawVec<T, A> as std::ops::Drop>::drop

Check 146: std::result::Result::<T, E>::unwrap_unchecked.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:1513:15 in function std::result::Result::<T, E>::unwrap_unchecked

Check 147: std::result::Result::<T, E>::unwrap_unchecked.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:1518:5 in function std::result::Result::<T, E>::unwrap_unchecked

Check 148: std::result::Result::<T, E>::unwrap_unchecked.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:1518:5 in function std::result::Result::<T, E>::unwrap_unchecked

Check 149: std::alloc::Layout::size.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:109:9 in function std::alloc::Layout::size

Check 150: std::alloc::Layout::size.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:109:9 in function std::alloc::Layout::size

Check 151: std::alloc::Layout::size.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:109:9 in function std::alloc::Layout::size

Check 152: std::alloc::Layout::size.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:109:9 in function std::alloc::Layout::size

Check 153: std::alloc::Layout::size.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:109:9 in function std::alloc::Layout::size

Check 154: std::alloc::Layout::size.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:109:9 in function std::alloc::Layout::size

Check 155: <std::slice::Iter<'a, T> as std::iter::Iterator>::any.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:241:27 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::any

Check 156: main::{closure#2}.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:17:58 in function main::{closure#2}

Check 157: main::{closure#2}.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:17:58 in function main::{closure#2}

Check 158: main::{closure#2}.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:17:58 in function main::{closure#2}

Check 159: main::{closure#2}.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:17:58 in function main::{closure#2}

Check 160: main::{closure#2}.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:17:58 in function main::{closure#2}

Check 161: main::{closure#2}.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:17:58 in function main::{closure#2}

Check 162: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:142:29 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 163: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:142:29 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 164: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:142:29 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 165: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:142:29 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 166: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:142:29 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 167: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:142:29 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 168: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:144:33 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 169: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:144:33 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 170: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:144:33 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 171: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:144:33 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 172: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:144:33 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 173: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:144:33 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 174: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.13
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:9 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 175: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.14
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:9 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 176: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.15
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:9 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 177: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.16
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:9 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 178: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.17
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:9 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 179: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.18
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:9 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 180: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.19
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:43 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 181: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.20
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:43 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 182: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.21
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:43 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 183: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.22
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:43 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 184: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.23
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:43 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 185: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.24
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:8:43 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 186: <std::slice::Iter<'a, T> as std::iter::Iterator>::next.pointer_dereference.25
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:147:25 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::next

Check 187: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:241:40 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 188: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:241:40 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 189: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:241:40 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 190: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:241:40 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 191: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:241:40 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 192: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:241:40 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 193: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:242:13 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 194: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:247:49 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 195: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:247:49 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 196: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:247:49 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 197: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:247:49 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 198: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:247:49 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 199: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.13
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:247:49 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 200: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.14
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:248:23 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 201: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.15
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:248:23 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 202: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.16
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:248:23 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 203: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.17
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:248:23 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 204: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.18
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:248:23 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 205: alloc::raw_vec::RawVec::<T, A>::current_memory.pointer_dereference.19
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:248:23 in function alloc::raw_vec::RawVec::<T, A>::current_memory

Check 206: <std::slice::Iter<'a, T> as std::iter::Iterator>::any.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:241:27 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::any

Check 207: <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2645:43 in function <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter

Check 208: <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2645:43 in function <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter

Check 209: <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2645:43 in function <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter

Check 210: <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2645:43 in function <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter

Check 211: <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2645:43 in function <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter

Check 212: <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2645:43 in function <std::vec::Vec<T, A> as std::iter::IntoIterator>::into_iter

Check 213: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 214: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 215: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 216: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 217: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 218: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 219: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 220: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 221: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 222: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 223: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 224: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 225: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.13
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 226: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.14
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 227: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.15
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 228: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.16
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 229: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.17
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 230: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.18
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 231: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.19
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 232: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.20
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 233: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.21
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 234: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.22
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 235: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.23
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 236: <std::option::Option<T> as std::cmp::PartialEq>::eq.pointer_dereference.24
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/option.rs:515:16 in function <std::option::Option<T> as std::cmp::PartialEq>::eq

Check 237: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:30 in function std::slice::Iter::<'a, T>::post_inc_start

Check 238: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:30 in function std::slice::Iter::<'a, T>::post_inc_start

Check 239: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:30 in function std::slice::Iter::<'a, T>::post_inc_start

Check 240: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:30 in function std::slice::Iter::<'a, T>::post_inc_start

Check 241: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:30 in function std::slice::Iter::<'a, T>::post_inc_start

Check 242: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:30 in function std::slice::Iter::<'a, T>::post_inc_start

Check 243: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:17 in function std::slice::Iter::<'a, T>::post_inc_start

Check 244: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:17 in function std::slice::Iter::<'a, T>::post_inc_start

Check 245: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:17 in function std::slice::Iter::<'a, T>::post_inc_start

Check 246: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:17 in function std::slice::Iter::<'a, T>::post_inc_start

Check 247: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:17 in function std::slice::Iter::<'a, T>::post_inc_start

Check 248: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:67:17 in function std::slice::Iter::<'a, T>::post_inc_start

Check 249: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.13
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:88:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 250: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.14
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:88:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 251: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.15
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:88:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 252: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.16
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:88:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 253: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.17
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:88:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 254: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.18
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:88:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 255: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.19
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:90:31 in function std::slice::Iter::<'a, T>::post_inc_start

Check 256: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.20
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:90:31 in function std::slice::Iter::<'a, T>::post_inc_start

Check 257: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.21
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:90:31 in function std::slice::Iter::<'a, T>::post_inc_start

Check 258: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.22
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:90:31 in function std::slice::Iter::<'a, T>::post_inc_start

Check 259: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.23
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:90:31 in function std::slice::Iter::<'a, T>::post_inc_start

Check 260: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.24
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:90:31 in function std::slice::Iter::<'a, T>::post_inc_start

Check 261: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.25
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:64 in function std::slice::Iter::<'a, T>::post_inc_start

Check 262: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.26
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:64 in function std::slice::Iter::<'a, T>::post_inc_start

Check 263: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.27
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:64 in function std::slice::Iter::<'a, T>::post_inc_start

Check 264: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.28
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:64 in function std::slice::Iter::<'a, T>::post_inc_start

Check 265: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.29
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:64 in function std::slice::Iter::<'a, T>::post_inc_start

Check 266: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.30
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:64 in function std::slice::Iter::<'a, T>::post_inc_start

Check 267: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.31
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 268: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.32
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 269: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.33
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 270: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.34
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 271: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.35
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 272: std::slice::Iter::<'a, T>::post_inc_start.pointer_dereference.36
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:93:21 in function std::slice::Iter::<'a, T>::post_inc_start

Check 273: <std::vec::Vec<T, A> as std::ops::Deref>::deref.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2500:55 in function <std::vec::Vec<T, A> as std::ops::Deref>::deref

Check 274: <std::vec::Vec<T, A> as std::ops::Deref>::deref.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2500:55 in function <std::vec::Vec<T, A> as std::ops::Deref>::deref

Check 275: <std::vec::Vec<T, A> as std::ops::Deref>::deref.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2500:55 in function <std::vec::Vec<T, A> as std::ops::Deref>::deref

Check 276: <std::vec::Vec<T, A> as std::ops::Deref>::deref.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2500:55 in function <std::vec::Vec<T, A> as std::ops::Deref>::deref

Check 277: <std::vec::Vec<T, A> as std::ops::Deref>::deref.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2500:55 in function <std::vec::Vec<T, A> as std::ops::Deref>::deref

Check 278: <std::vec::Vec<T, A> as std::ops::Deref>::deref.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2500:55 in function <std::vec::Vec<T, A> as std::ops::Deref>::deref

Check 279: <&mut I as std::iter::ExactSizeIterator>::len.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/exact_size.rs:142:9 in function <&mut I as std::iter::ExactSizeIterator>::len

Check 280: <&mut I as std::iter::ExactSizeIterator>::len.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/exact_size.rs:142:9 in function <&mut I as std::iter::ExactSizeIterator>::len

Check 281: <&mut I as std::iter::ExactSizeIterator>::len.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/exact_size.rs:142:9 in function <&mut I as std::iter::ExactSizeIterator>::len

Check 282: <&mut I as std::iter::ExactSizeIterator>::len.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/exact_size.rs:142:9 in function <&mut I as std::iter::ExactSizeIterator>::len

Check 283: <&mut I as std::iter::ExactSizeIterator>::len.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/exact_size.rs:142:9 in function <&mut I as std::iter::ExactSizeIterator>::len

Check 284: <&mut I as std::iter::ExactSizeIterator>::len.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/exact_size.rs:142:9 in function <&mut I as std::iter::ExactSizeIterator>::len

Check 285: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:52 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 286: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:52 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 287: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:52 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 288: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:52 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 289: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:52 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 290: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:52 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 291: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:63 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 292: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:63 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 293: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:63 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 294: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:63 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 295: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:63 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 296: std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/cmp.rs:1314:63 in function std::cmp::impls::<impl std::cmp::PartialEq for usize>::eq

Check 297: <std::vec::Vec<T, A> as std::ops::Drop>::drop.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2888:81 in function <std::vec::Vec<T, A> as std::ops::Drop>::drop

Check 298: <std::vec::Vec<T, A> as std::ops::Drop>::drop.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2888:81 in function <std::vec::Vec<T, A> as std::ops::Drop>::drop

Check 299: <std::vec::Vec<T, A> as std::ops::Drop>::drop.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2888:81 in function <std::vec::Vec<T, A> as std::ops::Drop>::drop

Check 300: <std::vec::Vec<T, A> as std::ops::Drop>::drop.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2888:81 in function <std::vec::Vec<T, A> as std::ops::Drop>::drop

Check 301: <std::vec::Vec<T, A> as std::ops::Drop>::drop.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2888:81 in function <std::vec::Vec<T, A> as std::ops::Drop>::drop

Check 302: <std::vec::Vec<T, A> as std::ops::Drop>::drop.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:2888:81 in function <std::vec::Vec<T, A> as std::ops::Drop>::drop

Check 303: std::ptr::drop_in_place.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 304: std::ptr::drop_in_place.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 305: std::ptr::drop_in_place.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 306: std::ptr::drop_in_place.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 307: std::ptr::drop_in_place.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 308: std::ptr::drop_in_place.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 309: std::ptr::drop_in_place.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 310: std::ptr::drop_in_place.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 311: std::ptr::drop_in_place.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 312: std::ptr::drop_in_place.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 313: std::ptr::drop_in_place.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 314: std::ptr::drop_in_place.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/mod.rs:487:1 in function std::ptr::drop_in_place

Check 315: std::vec::Vec::<T, A>::len.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:1914:9 in function std::vec::Vec::<T, A>::len

Check 316: std::vec::Vec::<T, A>::len.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:1914:9 in function std::vec::Vec::<T, A>::len

Check 317: std::vec::Vec::<T, A>::len.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:1914:9 in function std::vec::Vec::<T, A>::len

Check 318: std::vec::Vec::<T, A>::len.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:1914:9 in function std::vec::Vec::<T, A>::len

Check 319: std::vec::Vec::<T, A>::len.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:1914:9 in function std::vec::Vec::<T, A>::len

Check 320: std::vec::Vec::<T, A>::len.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/mod.rs:1914:9 in function std::vec::Vec::<T, A>::len

Check 321: std::mem::ManuallyDrop::<T>::take.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/mem/manually_drop.rs:113:18 in function std::mem::ManuallyDrop::<T>::take

Check 322: std::mem::ManuallyDrop::<T>::take.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/mem/manually_drop.rs:113:18 in function std::mem::ManuallyDrop::<T>::take

Check 323: std::mem::ManuallyDrop::<T>::take.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/mem/manually_drop.rs:113:18 in function std::mem::ManuallyDrop::<T>::take

Check 324: std::mem::ManuallyDrop::<T>::take.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/mem/manually_drop.rs:113:18 in function std::mem::ManuallyDrop::<T>::take

Check 325: std::mem::ManuallyDrop::<T>::take.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/mem/manually_drop.rs:113:18 in function std::mem::ManuallyDrop::<T>::take

Check 326: std::mem::ManuallyDrop::<T>::take.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/mem/manually_drop.rs:113:18 in function std::mem::ManuallyDrop::<T>::take

Check 327: main::{closure#3}.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:19:58 in function main::{closure#3}

Check 328: main::{closure#3}.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:19:58 in function main::{closure#3}

Check 329: main::{closure#3}.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:19:58 in function main::{closure#3}

Check 330: main::{closure#3}.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:19:58 in function main::{closure#3}

Check 331: main::{closure#3}.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:19:58 in function main::{closure#3}

Check 332: main::{closure#3}.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: tests/bookrunner/books/Rust by Example/Functions/Closures/Examples in std/Iterator::any/22.rs:19:58 in function main::{closure#3}

Check 333: std::alloc::Layout::align.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:119:9 in function std::alloc::Layout::align

Check 334: std::alloc::Layout::align.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:119:9 in function std::alloc::Layout::align

Check 335: std::alloc::Layout::align.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:119:9 in function std::alloc::Layout::align

Check 336: std::alloc::Layout::align.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:119:9 in function std::alloc::Layout::align

Check 337: std::alloc::Layout::align.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:119:9 in function std::alloc::Layout::align

Check 338: std::alloc::Layout::align.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/alloc/layout.rs:119:9 in function std::alloc::Layout::align

Check 339: <std::result::Result<T, F> as std::ops::FromResidual<std::result::Result<std::convert::Infallible, E>>>::from_residual.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/result.rs:2125:23 in function <std::result::Result<T, F> as std::ops::FromResidual<std::result::Result<std::convert::Infallible, E>>>::from_residual

Check 340: std::iter::ExactSizeIterator::len.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/macros/mod.rs:45:85 in function std::iter::ExactSizeIterator::len

Check 341: std::boxed::Box::<T, A>::into_unique.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1142:30 in function std::boxed::Box::<T, A>::into_unique

Check 342: std::boxed::Box::<T, A>::into_unique.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1142:30 in function std::boxed::Box::<T, A>::into_unique

Check 343: std::boxed::Box::<T, A>::into_unique.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1142:30 in function std::boxed::Box::<T, A>::into_unique

Check 344: std::boxed::Box::<T, A>::into_unique.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1142:30 in function std::boxed::Box::<T, A>::into_unique

Check 345: std::boxed::Box::<T, A>::into_unique.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1142:30 in function std::boxed::Box::<T, A>::into_unique

Check 346: std::boxed::Box::<T, A>::into_unique.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/boxed.rs:1142:30 in function std::boxed::Box::<T, A>::into_unique

Check 347: std::ptr::NonNull::<T>::as_ref.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/non_null.rs:381:20 in function std::ptr::NonNull::<T>::as_ref

Check 348: std::ptr::NonNull::<T>::as_ref.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/non_null.rs:381:20 in function std::ptr::NonNull::<T>::as_ref

Check 349: std::ptr::NonNull::<T>::as_ref.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/non_null.rs:381:20 in function std::ptr::NonNull::<T>::as_ref

Check 350: std::ptr::NonNull::<T>::as_ref.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/non_null.rs:381:20 in function std::ptr::NonNull::<T>::as_ref

Check 351: std::ptr::NonNull::<T>::as_ref.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/non_null.rs:381:20 in function std::ptr::NonNull::<T>::as_ref

Check 352: std::ptr::NonNull::<T>::as_ref.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ptr/non_null.rs:381:20 in function std::ptr::NonNull::<T>::as_ref

Check 353: alloc::raw_vec::RawVec::<T, A>::ptr.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:224:9 in function alloc::raw_vec::RawVec::<T, A>::ptr

Check 354: alloc::raw_vec::RawVec::<T, A>::ptr.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:224:9 in function alloc::raw_vec::RawVec::<T, A>::ptr

Check 355: alloc::raw_vec::RawVec::<T, A>::ptr.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:224:9 in function alloc::raw_vec::RawVec::<T, A>::ptr

Check 356: alloc::raw_vec::RawVec::<T, A>::ptr.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:224:9 in function alloc::raw_vec::RawVec::<T, A>::ptr

Check 357: alloc::raw_vec::RawVec::<T, A>::ptr.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:224:9 in function alloc::raw_vec::RawVec::<T, A>::ptr

Check 358: alloc::raw_vec::RawVec::<T, A>::ptr.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/raw_vec.rs:224:9 in function alloc::raw_vec::RawVec::<T, A>::ptr

Check 359: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:12 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 360: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:12 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 361: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:12 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 362: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:12 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 363: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:12 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 364: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:12 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 365: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:36 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 366: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:36 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 367: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:36 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 368: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:36 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 369: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:36 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 370: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:149:36 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 371: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.13
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:46 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 372: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.14
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:46 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 373: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.15
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:46 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 374: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.16
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:46 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 375: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.17
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:46 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 376: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.18
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:46 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 377: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.19
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 378: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.20
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 379: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.21
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 380: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.22
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 381: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.23
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 382: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.24
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:155:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 383: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.25
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:160:23 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 384: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.26
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:160:23 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 385: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.27
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:160:23 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 386: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.28
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:160:23 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 387: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.29
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:160:23 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 388: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.30
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:160:23 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 389: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.31
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:161:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 390: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.32
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:161:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 391: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.33
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:161:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 392: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.34
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:161:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 393: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.35
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:161:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 394: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.36
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:161:13 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 395: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.37
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:163:27 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 396: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.38
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:163:27 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 397: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.39
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:163:27 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 398: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.40
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:163:27 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 399: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.41
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:163:27 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 400: <std::vec::IntoIter<T, A> as std::iter::Iterator>::next.pointer_dereference.42
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/alloc/src/vec/into_iter.rs:163:27 in function <std::vec::IntoIter<T, A> as std::iter::Iterator>::next

Check 401: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.1
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 402: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.2
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 403: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.3
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 404: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.4
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 405: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.5
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 406: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.6
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 407: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.7
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 408: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.8
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 409: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.9
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 410: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.10
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 411: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.11
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 412: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.12
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 413: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.13
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 414: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.14
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 415: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.15
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 416: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.16
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 417: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.17
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 418: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.18
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 419: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.19
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 420: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.20
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 421: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.21
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 422: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.22
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 423: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.23
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 424: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.24
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 425: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.25
- Status: SUCCESS
- Description: "dereference failure: pointer NULL"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 426: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.26
- Status: SUCCESS
- Description: "dereference failure: pointer invalid"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 427: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.27
- Status: SUCCESS
- Description: "dereference failure: deallocated dynamic object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 428: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.28
- Status: SUCCESS
- Description: "dereference failure: dead object"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 429: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.29
- Status: SUCCESS
- Description: "dereference failure: pointer outside object bounds"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 430: <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq.pointer_dereference.30
- Status: SUCCESS
- Description: "dereference failure: invalid integer address"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/ops/control_flow.rs:82:30 in function <std::ops::ControlFlow<B, C> as std::cmp::PartialEq>::eq

Check 431: <std::slice::Iter<'a, T> as std::iter::Iterator>::any.unwind.0
- Status: SUCCESS
- Description: "unwinding assertion loop 0"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/slice/iter/macros.rs:241:17 in function <std::slice::Iter<'a, T> as std::iter::Iterator>::any

Check 432: std::iter::Iterator::try_fold.unwind.0
- Status: SUCCESS
- Description: "unwinding assertion loop 0"
- Location: ~/.rustup/toolchains/nightly-2022-06-09-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/iter/traits/iterator.rs:2189:9 in function std::iter::Iterator::try_fold


SUMMARY:
** 0 of 432 failed

VERIFICATION:- SUCCESSFUL


Verification Time: 0.8322282s
              </xmp>
            </div><!-- class="output-box" -->
            
          

          
          <div class="output-box"><xmp>stderr:
            
warning: this method call resolves to `<&[T; N] as IntoIterator>::into_iter` (due to backwards compatibility), but will resolve to <[T; N] as IntoIterator>::into_iter in Rust 2021
--> 22.rs:19:40
|
19 |     println!("2 in array2: {}", array2.into_iter().any(|&x| x == 2));
|                                        ^^^^^^^^^
|
= note: `#[warn(array_into_iter)]` on by default
= warning: this changes meaning in Rust 2021
= note: for more information, see <https://doc.rust-lang.org/nightly/edition-guide/rust-2021/IntoIterator-for-arrays.html>
help: use `.iter()` instead of `.into_iter()` to avoid ambiguity
|
19 |     println!("2 in array2: {}", array2.iter().any(|&x| x == 2));
|                                        ~~~~
help: or use `IntoIterator::into_iter(..)` instead of `.into_iter()` to explicitly iterate by value
|
19 |     println!("2 in array2: {}", IntoIterator::into_iter(array2).any(|&x| x == 2));
|                                 ++++++++++++++++++++++++      ~

warning: 1 warning emitted

            </xmp>
          </div><!-- class="output-box" -->
          

        </div><!-- class="command-content" -->
      </div><!-- class="fail" -->
    </div><!-- id="command-table" -->
    

  </div><!-- class="stage" -->
  


</div><!-- id="content" -->
</body>
</html>
